src/main/java/me/coley/cafedude/instruction/InvokeDynamicInstruction.java

```diff
@@ -0,0 +1,132 @@
+package me.coley.cafedude.instruction;
+
+import me.coley.cafedude.constant.ConstPoolEntry;
+
+import java.util.Arrays;
+
+/**
+ * InvokeDynamic instruction.
+ *
+ * @author xDark
+ */
+public class InvokeDynamicInstruction extends BasicInstruction {
+	private String name;
+	private String desc;
+	private MethodHandle methodHandle;
+	private ConstPoolEntry[] bootstrapArguments;
+
+	/**
+	 * @param name
+	 * 		Method name.
+	 * @param desc
+	 * 		Method desc.
+	 * @param methodHandle
+	 * 		Bootstrap method.
+	 * @param bootstrapArguments
+	 * 		Bootstrap arguments.
+	 */
+	public InvokeDynamicInstruction(String name, String desc, MethodHandle methodHandle, ConstPoolEntry[] bootstrapArguments) {
```

Comment:
For cases like this, how are you going to do write-back to `byte[]`?

I don't see how the instruction will be able to map these values to the values in the constant pool without doing a search for a matching value. Most of the library currently just does CP indices where you then would do something like: `pool.getCpEntry(thing.getNameIndex())`